content/behind-atom/sections/developing-node-modules.md

```diff
@@ -7,15 +7,15 @@ Atom contains a number of packages that are Node modules instead of Atom package
 
 #### Linking a Node Module Into Your Atom Dev Environment
 
-Here are the steps to run a local version of a node module *not an apm* within Atom. We're using `atom-keymap` as an example:
+Here are the steps to run a local version of a node module within Atom. We're using `atom-keymap` as an example:
```

Comment:
I think the "not an apm" wording was weird and awkward, but I wonder if it's worth highlighting that we're talking about a Node module here, and not an Atom package?